tests/foreman/cli/test_host.py

```diff
@@ -627,12 +603,9 @@ def test_negative_register_twice(module_ak_with_cv, module_org, rhel7_contenthos
 
     :CaseLevel: System
     """
-    rhel7_contenthost.install_katello_ca(target_sat)
-    rhel7_contenthost.register_contenthost(module_org.label, module_ak_with_cv.name)
+    rhel7_contenthost.register(module_org, module_ak_with_cv.name, target_sat)
     assert rhel7_contenthost.subscribed
-    result = rhel7_contenthost.register_contenthost(
-        module_org.label, module_ak_with_cv.name, force=False
-    )
+    result = rhel7_contenthost.register(module_org, module_ak_with_cv.name, target_sat, force=False)
```

Comment:
I think it would be better to test this scenario against all content host versions, as RC could change for double registration?

Reply:
as of now register requires location as a second param, but after https://github.com/SatelliteQE/robottelo/pull/11783 gets in, you could omit it by doing `rhel7_contenthost.register(module_org, None, module_ak_with_cv.name, target_sat, force=False)`

Reply:
Yes. Updated.